dashboard/config/openbsd/recreate.sh

```diff
@@ -31,6 +31,7 @@ SYZ_DIR="$(cd "$(dirname "${0}")"/../../..; pwd -P)"
 
 ZONE=us-central1-b
 INSTANCE=ci-openbsd
+HOST="${HOST:-ci-openbsd}"
```

Comment:
Nit: should it default to ci-openbsd or ${INSTANCE}?

Reply:
Updated. PTAL.